doc/trapezoidal_industrial_planner/trapezoidal_industrial_planner.rst

```diff
@@ -0,0 +1,371 @@
+   ``moveit_msgs::MotionPlanRequest``
+-  ``trajectory/joint_trajectory/joint_names``: a list of the joint
+   names of the generated joint trajectory
+-  ``trajectory/joint_trajectory/points/(positions,velocities,accelerations,time_from_start)``:
+   a list of generated way points. Each point has
+   positions/velocities/accelerations of all joints (same order as the
+   joint names) and time from start. The last point will have zero
+   velocity and acceleration.
+-  ``group_name``: name of the planning group
+-  ``error_code/val``: error code of the motion planning
+
+The LIN motion command
+----------------------
+
+This planner generates linear Cartesian trajectory between goal and
+start poses. The planner uses the Cartesian limits to generate a
+trapezoidal velocity profile in Cartesian space. The translational
+motion is a linear interpolation between start and goal position vector.
+The rotational motion is quaternion slerp between start and goal
+orientation. The translational and rotational motion is synchronized in
+time. This planner only accepts start state with zero velocity. Planning
+result is a joint trajectory. The user needs to adapt the Cartesian
+velocity/acceleration scaling factor if motion plan fails due to
+violation of joint space limits.
+
+Input parameters in ``moveit_msgs::MotionPlanRequest``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  ``planner_id``: LIN
+-  ``group_name``: name of the planning group
+-  ``max_velocity_scaling_factor``: scaling factor of maximal Cartesian
+   translational/rotational velocity
+-  ``max_acceleration_scaling_factor``: scaling factor of maximal
+   Cartesian translational/rotational acceleration/deceleration
+-  ``start_state/joint_state/(name, position and velocity``: joint
+   name/position of the start state.
+-  ``goal_constraints`` (goal can be given in joint space or Cartesian
+   space)
+
+   -  for goal in joint space
+
+      -  ``goal_constraints/joint_constraints/joint_name``: goal joint
+         name
+      -  ``goal_constraints/joint_constraints/position``: goal joint
+         position
+
+   -  for goal in Cartesian space
+
+      -  ``goal_constraints/position_constraints/header/frame_id``:
+         frame this data is associated with
+      -  ``goal_constraints/position_constraints/link_name``: target
+         link name
+      -  ``goal_constraints/position_constraints/constraint_region``:
+         bounding volume of the target point
+      -  ``goal_constraints/position_constraints/target_point_offset``:
+         offset (in the link frame) for the target point on the target
+         link (optional)
+
+-  max\_velocity\_scaling\_factor: rescale the maximal velocity
+-  max\_acceleration\_scaling\_factor: rescale the maximal acceleration
+
+planning results in ``moveit_msg::MotionPlanResponse``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  ``trajectory_start``: bypass the ``start_state`` in
```

Comment:
I'm not sure I understand this one.

Reply:
Me neiter ;-) I'll have to find it out in the code. Probably the input state from the request is copied over to the response...